src/main/kotlin/org/octopusden/octopus/oc/template/plugins/gradle/service/diagnostics/PostMortemAnalyzer.kt

```
@@ -0,0 +1,311 @@
+                result += parseFlatJsonObject(trimmed)
+            }
+        }
+        return result
+    }
+
+    internal fun parseMetaJson(file: File): Map<String, String> {
+        if (!file.exists()) return emptyMap()
+        return parseFlatJsonObject(file.readText().trim())
+    }
+
+    internal fun parseFlatJsonObject(src: String): Map<String, String> {
```

detekt (1.23.8)

[warning] The function parseFlatJsonObject appears to be too complex based on Cyclomatic Complexity (complexity: 52). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)